web/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/StyleSheetBuilder.kt

```diff
@@ -241,66 +242,54 @@ interface SelectorsScope {
     fun slotted(selector: CSSSelector): CSSSelector = PseudoElementInternal.Slotted(selector)
 }
 
-private data class Id(val id: String) : CSSSelector() {
-    override fun toString(): String = "#$id"
-}
-
-private data class Type(val type: String) : CSSSelector() {
-    override fun toString(): String = type
-}
-
-private object Universal : CSSSelector() {
-    override fun toString(): String = "*"
-}
-
-private data class Raw(val selector: String) : CSSSelector() {
+private data class RawSelector(val selector: String) : CSSSelector() {
     override fun toString(): String = selector
 }
 
 private data class Combine(val selectors: MutableList<CSSSelector>) : CSSSelector() {
-    override fun contains(other: CSSSelector, strict: Boolean): Boolean =
-        contains(this, other, selectors, strict)
```

Comment:
just to be sure: was `strict` unused at all?

Reply:
@eymar correct, it was never used and was true in 100% of cases (actually there's a single point where it was used) - and keeping something just in case is a wrong thing to do (it also was quite confusing)